Source/WPF/MyMoney/Database/Money.cs

```diff
@@ -2635,6 +2636,13 @@ private int OnlineAccountId
             set { this.onlineAccountId = value; }
         }
 
+        [DataMember]

```

Comment:
I don't think this needs to be a DataMember or stored in the database.  It should just be a cache.